flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -215,5 +218,89 @@
                                     + "faster updating metrics. Increase this value if the metric fetcher causes too much load. Setting this value to 0 "
                                     + "disables the metric fetching completely.");
 
+    /** Controls which job status metrics will be exposed. */
+    public static final ConfigOption<List<JobStatusMetrics>> JOB_STATUS_METRICS =
+            key("metrics.job.status.enable")
```

Comment:
this is just to reduce cardinality or is there any other reason?

Reply:
> to reduce cardinality

yup.